### ISSUE TITLE
Missing notebook

### ISSUE THREAD

**Issue by jsrodrigo** [langchain-ai/report-mAIstro#2]
Great workflow and documentation! Would be great if you can upload the notebook too

**Comment by LeonidShamis** [comment]
Yes, some links in the [README](https://github.com/langchain-ai/report-mAIstro/blob/main/README.md) are broken either due to missing code or a wrong link, for example:

- [notebook](https://github.com/langchain-ai/report-mAIstro/blob/main/ntbk/report_maistro.ipynb) - missing the entire `ntbk` folder with the notebook(s) example(s)
- [example reports](https://github.com/langchain-ai/report-mAIstro/blob/main/report_examples/reports) - points to a missing `reports` subdirectory under the `report_examples`

**Comment by SushantKadam73** [comment]
i am unable to run the files

**Comment by katnoria** [comment]
You can find the notebook in `test-updated-prompts` branch. 
https://github.com/langchain-ai/report-mAIstro/blob/rlm/test-updated-prompts/ntbk/report_maistro.ipynb

**Comment by supernitin** [comment]
@katnoria How did you find this? Is this somewhere we are supposed to know to look for notebooks?

**Comment by rlancemartin** [comment]
Added!
https://github.com/langchain-ai/report-mAIstro/blob/main/src/report_maistro/graph.ipynb
